### PR TITLE
Test for SYCL2020 `host_accessor` and `accessor` property 

### PR DESCRIPTION
**Pull request by andreyromanof**
This provides tests for `no_init` property of `host_accessor` and `accessor`.

Depends on:
- #325 

[Link](https://github.com/KhronosGroup/SYCL-CTS/pull/327/commits/b88fc672e8b385fc60530fe45a55b7472c64c578) to related changes

### CODE REVIEW

File: tests/accessor/generic_accessor_properties_fp16.cpp [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+//
+*******************************************************************************/
+
+#include "../common/common.h"
+
+// FIXME: re-enable when sycl::accessor is implemented
+#if !defined(__HIPSYCL__) && !defined(__COMPUTECPP__) && \
+    !defined(__SYCL_COMPILER_VERSION)
+#include "accessor_common.h"
+#include "generic_accessor_properties.hpp"
```
Comment (yuriykoch) [inline tests/accessor/generic_accessor_properties_fp16.cpp:15]: Looks like we've started adding `.hpp` files at some point in tests for accessors. Let's align with the rest of the code base and use `.h` extension. Currently we don't even have `clangformat.py` enabled for `hpp` files.

Comment (andreyromanof) [reply tests/accessor/generic_accessor_properties_fp16.cpp]: Renamed

**Review by keryell — APPROVED** [review]
Thanks!

**Comment by bader** [comment]
@andreyromanof, please, resolve merge conflicts.

**Comment by andreyromanof** [comment]
Conflicts resolved.

**Pull request merged**